lib/canohost.c

```diff
@@ -0,0 +1,140 @@
+  /********************************************************************\
+  * BitlBee -- An IRC to other IM-networks gateway                     *
+  *                                                                    *
+  * Copyright 2002-2017 Wilmer van der Gaast and others                *
+  \********************************************************************/
+
+/*
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
```

Comment:
This *program*, not this file. It's the default file header, I flipped a coin and it told me to keep it. ;-)